docs/CUSTOMIZING.rst

```diff
@@ -53,6 +120,22 @@ In order to disable redaction controls for certain metadata fields, you can add
 
 Note that fields specified in ``no_redact_control_keys`` will have redaction disabled on all image formats. If you wish to disable redaction of a metadata field on, for example, Aperio images only, you can add that metadata key and expected value to the ``no_redact_control_keys_format_aperio`` dictionary.
 
+.. code-block:: python
+
+  [wsi_deid]
+  ...
+  no_redact_control_keys = {
+      "^internal;aperio_version$": "",
+      "^internal;openslide;openslide\.(?!comment$)": "",
+      "^internal;openslide;tiff\.(ResolutionUnit|XResolution|YResolution)$": "^\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?)(\s*,\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?))*\s*$",
+      "^internal;openslide;tiff\.ResolutionUnit": ""}
+  no_redact_control_keys_format_aperio = {
+      "^internal;openslide;aperio\.(AppMag|MPP|Exposure (Time|Scale))$": "^\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?)(\s*,\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?))*\s*$"}
+  no_redact_control_keys_format_hamamatsu = {
+      "^internal;openslide;hamamatsu\.SourceLens$": "^\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?)(\s*,\s*[+-]?(\d+([.]\d*)?([eE][+-]?\d+)?|[.]\d+([eE][+-]?\d+)?))*\s*$"}
+  no_redact_control_keys_format_philips = {}
```

Comment:
Let's add the isyntax line (`no_redact_control_keys_format_isyntax = {}`)

Reply:
be52e04